packages/golden-layout/scss/goldenlayout-dark-theme.scss

```diff
@@ -362,7 +362,7 @@ body:not(.lm_dragging) .lm_header .lm_tab .lm_close_tab:hover {
 
 // If a specific Pane is maximized
 .lm_maximised {
-  border: 15px solid rgba($black, 0.85);
+  border: 15px solid background-opacity(80);
```

Comment:
Not sure exactly what's causing this, but maximise button is appearing black instead of gray:
![image](https://github.com/deephaven/web-client-ui/assets/4505624/69e4cdde-e72d-4f68-ae4f-65e75626e67e)

Reply:
This will probably need to be changed to use masks for coloring the icons.

Can't style inline svg with a css variable, it was being colored at compile time.